### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by DylanJ3191** [Tau-ri-Dev/JSGMod#1] [General]
So, when I try to build the MilkyWay part of JSGMod-1.18.2, it always fails. I was wondering if you could kindly put in the releases, a built version of the mod? It would help me a lot, thanks.

**Comment by DylanJ3191** [comment]
Oh, and the core mod will build for me, it's just the MilkyWay part that fails.

**Comment by VojtechSin** [comment]
Sorry, for delayed answer. 1.18.2 is not done and its abandoned for now
